docs/network-iot/hotspots-iot/data-only-onboarding.mdx

```diff
@@ -152,13 +151,13 @@ found.
 keypair = "/etc/helium_gateway/gateway_key.bin"
 ```
 
-It's advised to backup the gateway_key file, especially if it needs to be recovered after the
+It's advised to backup the `gateway_key.bin` file, especially if it needs to be recovered after the
 onboarding process.
 
 ### First Run
 
 With gateway-rs installed on the gateway, initialize the key for the Hotspot. If creating the
-gateway_key in /etc/helium_gateway/, `sudo` may be required on first run to create the key file.
+`gateway_key` in /etc/helium_gateway/, `sudo` may be required on first run to create the key file.
```

Comment:
we're just talking about the gateway key here so I don't think it needs a code wrapper. Trying to avoid overusing these to make reading the text easier.